docker/cli
# add --from-current flag to context create

**- What I did**

Added `--from-current` flag to `context create` command that simplifies creation of a context from the current context, acting as an alias for `--docker from-context=true` and automatically infers whether to create the context with kubernetes as per `--kubernetes from-context=true`

**- How I did it**
Uses the context store to export the current context, then updates the details (description and orchestrator), and uses the context store to import the new context.

**- How to verify it**
Tests have been added to check the new functionality and ensure the user cannot use the `--from-current` flag with other docker/kubernetes settings
